components/support/nimbus-fml/src/backends/mod.rs

```diff
@@ -55,32 +55,71 @@ pub trait CodeType {
     fn type_label(&self, oracle: &dyn CodeOracle) -> String;
 
     /// The language specific expression that gets a value of the `prop` from the `vars` object.
-    fn get_value(&self, oracle: &dyn CodeOracle, vars: &dyn Display, prop: &dyn Display) -> String {
+    fn property_getter(
```

Comment:
These defaults are all Kotlin specific. This won't do in the long run. How about changing the `identifiers` module in to `common`?